projects/laji/src/app/shared-modules/select/select/select.component.ts

```diff
@@ -65,7 +66,9 @@ export class SelectComponent<T extends idType|SelectOption = string> implements
       this.selected = [];
       this.open = false;
     }
-    this.initOptions(this.selected);
+    if (!this.selected || !this.selected.length || (this.selected.length !== this.selectedOptions.length)) {
+      this.initOptions(this.selected);
+    }
```

Comment:
capture what has changed on this `ngOnChanges` method (1st parameter in that). and then use that to decide if the `this.initOptions` should be called. (looks like it should be called at least when `options`, `checkboxType`, `disabled` and `selected` inputs change)

Reply:
@vixriihi i think i verified all conditions, but i don't understand if the condition has to be better presented